app/views/shared/_received-booking-card.html.erb

```diff
@@ -0,0 +1,20 @@
+<% @my_listings.each do |listing| %>
+  <% if listing.bookings.nil? %>
+    <p>Wow, such empty! 👶</p>
```

Comment:
Same as above. What if we add a call to action here? Something like:
<p>No bookings found.</p>
<p><%= link_to 'Create a new booking?', listings_path %></p>

Reply:
@worksofgenius My understanding was that an action here will be to create more received bookings, which I couldn't think of much more than to either 'Create more listings' (similar to what you suggested I think) or improve user profile to get more traction 😄